middleware/index.js

```diff
@@ -0,0 +1,227 @@
+'use strict';
+var chalk = require('chalk');
+var yeoman = require('yeoman-generator');
+
+var wsModels = require('loopback-workspace').models;
+
+var actions = require('../lib/actions');
+var helpers = require('../lib/helpers');
+var validateName = helpers.validateName;
+var async = require('async');
+
+function listChoices(items) {
+  var choices = [];
+  items.forEach(function(item, index) {
+    choices.push({
+      name: (index + 1) + '. ' + item,
+      value: item
+    });
+  });
+  return choices;
```

Comment:
Please simplify and use `items.map()` instead of `items.forEach()` + `choices.push()`.